bigtable/use-cases/fraudDetection/terraform/scripts/load_cbt.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+PROJECT_ID=$1
+REGION=$2
+CBT_INSTANCE=$3
+CBT_TABLE=$4
+GCS_BUCKET=$5
+
+echo "PROJECT_ID=$PROJECT_ID"
+echo "REGION=$REGION"
+echo "CBT_INSTANCE=$CBT_INSTANCE"
+echo "CBT_TABLE=$CBT_TABLE"
+echo "GCS_BUCKET=$GCS_BUCKET"
+
+mvn -f ../pom.xml compile exec:java -Dexec.mainClass=com.example.dataflowpipelines.GCSToCBT -Dexec.cleanupDaemonThreads=false \
```

Comment:
You might want to exclude this for now since it doesn't exist yet. Also, we should discuss if we need dataflow for loading in this data, it might be a lot faster to just do with some plain Java, but maybe if we use the local runner it would have the same effect – can discuss on next PR